src/lib/api/api.go

```diff
@@ -438,31 +437,38 @@ func (a *API) ModifyIntervalDuration(w http.ResponseWriter, r *http.Request) {
 	}
 	var req struct {
 		IntervalName string `json:"intervalName"`
-		Value        string `json:"value"`
+		Value        uint32 `json:"value"`
 	}
 	decoder := json.NewDecoder(r.Body)
 	if err := decoder.Decode(&req); err != nil {
-		panic(err)
+		a.ResponseStatus = "400"
+		a.Data = "Could not modify the config file. Value not a uint or intervalName is not a string"
```

Comment:
```suggestion
		a.Data = "Could not modify the config file"
```
and log the actual error as log.Error()